### PR TITLE
Improve buttons in call view.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk

### CODE REVIEW

File: Signal/src/view controllers/CallViewController.swift [hunk 1]
```diff
@@ -338,10 +338,10 @@ class CallViewController: UIViewController, CallDelegate {
...
         let button = UIButton()
         button.setImage(image, for:.normal)
-        button.imageEdgeInsets = UIEdgeInsetsMake(buttonInset(),
-                                                  buttonInset(),
-                                                  buttonInset(),
-                                                  buttonInset())
+        button.imageEdgeInsets = UIEdgeInsets(top: buttonInset(),
+                                              left: buttonInset(),
+                                              bottom: buttonInset(),
+                                              right: buttonInset())
```
Comment (charlesmchen) [inline Signal/src/view controllers/CallViewController.swift]: Style clean up.

File: Signal/src/view controllers/CallViewController.swift [hunk 2]
```diff
@@ -486,19 +486,12 @@ class CallViewController: UIViewController, CallDelegate {
...
-            } else {
-                // Show ongoing controls
-                let isOngoingCallControl = ongoingCallControls.contains(subview)
-                subview.isHidden = !isOngoingCallControl
-            }
-        }
+        incomingCallView.isHidden = !isRinging
+        incomingCallView.isUserInteractionEnabled = isRinging
+        ongoingCallView.isHidden = isRinging
+        ongoingCallView.isUserInteractionEnabled = !isRinging
```
Comment (charlesmchen) [inline Signal/src/view controllers/CallViewController.swift]: Fixes the "unresponsive hangup button" issue.

File: Signal/src/view controllers/CallViewController.swift [hunk 3]
```diff
@@ -549,6 +542,8 @@ class CallViewController: UIViewController, CallDelegate {
 
     func didPressTextMessage(sender speakerphoneButton: UIButton) {
         Logger.info("\(TAG) called \(#function)")
+
+        self.dismiss(animated: true)
```
Comment (charlesmchen) [inline Signal/src/view controllers/CallViewController.swift]: Is this right?  How do users return to an ongoing call?  How does the UI surface that there _is_ an ongoing call?

Comment (michaelkirk) [inline Signal/src/view controllers/CallViewController.swift]: that's the TODO 

=)

**Comment by michaelkirk** [comment]
LGTM bearing in mind that the "text message" feature is incomplete for now (no way to return to the call).

**Pull request merged**